build.gradle

```diff
@@ -0,0 +1,71 @@
+plugins {
+    id 'scala'
+    id 'maven-publish'
+    id 'com.github.maiflai.scalatest' version '0.15'
+}
+version '1.0.0'
+
+repositories {
+    mavenCentral()
+    maven {
```

Comment:
Yes, we can. But this section is for resolving your dependencies. You do not depend on any of our artifacts. For publishing you already defined our repo (see below).